packages/cosmos_ui_components/lib/components/template/cosmos_balance_card.dart

```diff
@@ -52,7 +54,7 @@ class CosmosBalanceCard extends StatelessWidget {
                 mainAxisAlignment: MainAxisAlignment.center,
                 children: [
                   Text(amountDisplayText, style: CosmosTextTheme.labelS),
-                  Text('available ${denomText.toUpperCase()}', style: CosmosTextTheme.copyMinus1Normal),
+                  Text('$secondaryText ${denomText.toUpperCase()}', style: CosmosTextTheme.copyMinus1Normal),
```

Comment:
what I meant here was to make the entire text a `secondaryText`, since in some languages the word order might be different.

```suggestion
                  Text(secondaryText, style: CosmosTextTheme.copyMinus1Normal),
```

Reply:
Updated. I made the whole `secondaryText` a required parameter.

Reply:
but it won't make much sense to be required, if you fall into case where its not displayed at all, maybe just make it default to `""`?

Reply:
Yeah you are right, my bad this should be optional. Just updated it